src/restaurants/restaurants.js

```diff
@@ -6,65 +8,71 @@ export function initializeRestaurants() {
   getRestaurants();
 }
 
-// export function getRestaurantsTemplate() {
-//   return `<h1>Choose a restaurant</h1>`;
-// }
+export function restaurantsTemplate() {
```

Comment:
Is there a reason for removing the get? Using get for naming suggests that it's a function returning the template, not a variable containing the template.